src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureDrawing,AreaCalculator {
```

Comment:
it would be better to have space here `implements FigureDrawing, AreaCalculator`